releases/sui/2025-01-ITS-v1.1.3.md

```diff
@@ -0,0 +1,127 @@
+# Sui ITS v1.1.3
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @blockchainguyy <ayush@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | - | TBD |
+| **Stagenet** | - | TBD |
+| **Testnet** | - | TBD |
+| **Mainnet** | - | TBD |
+
+[Release](https://github.com/axelarnetwork/axelar-cgp-sui/releases/tag/v1.1.3)
+
+[Changelog](https://github.com/axelarnetwork/axelar-cgp-sui/blob/v1.1.3/CHANGELOG.md)
+
+## Deployment
+
+- Ensure that ITS Hub has been upgraded to [v1.1.0](../cosmwasm/2025-01-ITS-v1.1.0.md) first.
+- The rollout involves deploying Sui ITS contracts version `v1.1.3` on Sui.
+
+```bash
+# Clone latest main and update deps
+npm ci
+```
+
+Create an `.env` config. Use `all` for `CHAINS` to run the cmd for every EVM chain, or set a specific chain.
+
+```yaml
+PRIVATE_KEY=<sui-deployer-key>
+PRIVATE_KEY_TYPE="mnemonic" # Optional
+SIGNATURE_SCHEME=secp256k1
+ENV=<devnet-amplifier|stagenet|testnet|mainnet>
+CHAIN=sui
+```
+
+### Deploy move contracts
+- Sync move directories
+```bash
+node sui/deploy-contract.js sync
+```
+
+- Delete existing gas `GasService` & `Example` config from `$ENV.json`
+
+- Deploy GasService contract
+```bash
+node sui/deploy-contract.js deploy GasService
+```
+
+- Transfer cap to Operators contract
+```bash
+node sui/operators.js storeCap
+```
+
+- Deploy InterchainTokenService and Example contract
+```bash
+node sui/deploy-contract.js deploy InterchainTokenService
+node sui/deploy-contract.js deploy Example
+```
+
+### Register Sui ITS on ITS Hub
+
+ITS hub contract configuration in json file must include the following attributes per chain:
+```bash
+"axelar": {
+  "contracts": {
+    ...
+    "InterchainTokenService": {
+      ...
+      "some-sui-chain": {
+        "maxUintBits": 64,
+        "maxDecimalsWhenTruncating": 8
```

Comment:
thought about the config again, let's switch to using `6` for the new deployment so popular meme coins with low unit dollar value don't hit max transfer limit